### ISSUE TITLE
Remove ultra beast from tournaments

### ISSUE THREAD

**Issue by RatedPG21** [SmashMC-Development/Suggestions#963]
**Describe the suggestion**
Ultra beasts should be removed from the random tournaments that happen every 3 hours because they are overpowered 

**Which servers does this apply to?**
All

**Comment by jantjewieger** [comment]
I do not agree with this. There is a chance to get them yourself of course as well. And after all they are all natdex (except for naganadel). But even if someone has one u can try and play around it. Having an UB doesnt give u a guaranteed win and therefore id say to keep them in there
